stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -352,6 +353,7 @@ private void executeTokenTask(Executor executor, AsyncTask<Void, Void, ResponseW
         cardParams.put("address_zip", StripeTextUtils.nullIfBlank(card.getAddressZip()));
         cardParams.put("address_state", StripeTextUtils.nullIfBlank(card.getAddressState()));
         cardParams.put("address_country", StripeTextUtils.nullIfBlank(card.getAddressCountry()));
+        cardParams.put("funding", StripeTextUtils.nullIfBlank(card.getFunding()));
```

Comment:
You don't want to put the `funding` property in your request to the Stripe API - this is something that should be present in the response, but not in the request. https://stripe.com/docs/api#create_card_token-card